components/plurals/src/operands.rs

```diff
@@ -15,6 +15,19 @@ use std::str::FromStr;
 ///
 /// See [full operands description](http://unicode.org/reports/tr35/tr35-numbers.html#Operands).
 ///
+/// # Data Types
+///
+/// The following types can be converted to PluralOperands:
```

Comment:
This doc is on PluralOperands itself.  Is it good practice to link to itself?

Reply:
Hmm, I don't know if I have an opinion about whether or not it would be "good practice." 

I prefer it, because it highlights the type. To me, that makes it more apparent when I"m reading docs that this text is important and meaningful in the code in some way.

Functionally, it does nothing. Clicking on it just keeps you on the same page. 

I am happy either way, but I generally approach docs with a link-everything mentality because I think it looks cleaner myself.